test/lib/absinthe/logger_test.exs

```diff
@@ -0,0 +1,71 @@
+defmodule Absinthe.LoggerTest do
+  use Absinthe.Case, async: true
+
+  describe "Absinthe.Logger.filter_variables/1" do
+    @value "abcd"
+    @variables %{"token" => @value, "password" => @value, "alsoUnsafe" => @value}
+    @filtered "[FILTERED]"
+
+    test "it filters the set values, with defaults" do
+      assert %{
+        "token" => @filtered,
+        "password" => @filtered,
+        "alsoUnsafe" => @value
+      } = Absinthe.Logger.filter_variables(@variables)
+      # Change the config
+      Application.put_env(:absinthe, Absinthe.Logger, [filter_variables: ~w(token alsoUnsafe)])
```

Comment:
It will, which is annoying. I need to find a better pattern for testing against application configuration.